dyn_em/module_initialize_real.F

```diff
@@ -69,6 +69,8 @@ SUBROUTINE init_domain_rk ( grid &
    )
 
       USE module_optional_input
+      USE module_radiation_driver
```

Comment:
may want only cal_cldfra3 here

Reply:
Agree that makes good sense.  Change made and pushed.